getyourdata/organization/tests.py

```diff
@@ -488,6 +513,19 @@ def test_moderator_can_review_and_ignore_organization_draft(self):
 @isSeleniumTest()
 class OrganizationListJavascriptTests(LiveServerTestCase):
     def setUp(self):
+        self.xpaths = ["(//input[@type='checkbox' and contains(@onclick, 'orgList')])[1]",
```

Comment:
Not a very good idea; if the user wants to know what element some line is searching for, instead of the element being described in the same line, it's described here, causing needless jumping back and forth in the code.